src/edge/edgeManager.ts

```diff
@@ -483,6 +511,10 @@ export class EdgeManager {
                 label: Constants.EXISTING_MODULE,
                 description: Constants.EXISTING_MODULE_DESCRIPTION,
             },
+            {
+                label: Constants.STREAM_ANALYTICS,

```

Comment:
I had a offline sync with Xin. Can you rearrange the order to put ASA module below C# Function module? So the new order will be:
* C# Module
* Node.js Module
* Python Module
* Azure Functions - C#
* Azure Stream Analytics
* Existing Module on ACR
* Existing Module